src/main.rs

```diff
@@ -220,6 +226,26 @@ fn quickadd(matches: &ArgMatches, text: String) -> Result<String, String> {
     Ok(projects::green_string("✓"))
 }
 
+#[cfg(not(tarpaulin_include))]
+fn task_edit(matches: &ArgMatches) -> Result<String, String> {
+    let config = fetch_config(matches)?;
+    let project_name = fetch_project(matches, &config)?;
+    let project_id = projects::project_id(&config, &project_name)?;
+    let project_tasks = request::items_for_project(&config, &project_id)?;
+
+    let selected_task = config::select_input("Choose a task of the project:", project_tasks)?;
+    let task_content = selected_task.content.as_str();
+
+    let new_task_content =
+        config::get_input_with_default("Edit the task you selected:", task_content)?;
+
+    if task_content == new_task_content {
+        return Ok(String::from(""));
```

Comment:
I think that everything from line 233 to 246 can be moved to the `projects`module and named something like "rename_item"

The reason being is that we want to keep the code in the main module as small as possible (it just gets the necessary values from matches) and then do the testing in the other modules where we won't have to construct the matches.